### DISCUSSION TITLE
weather warnings - which location is it?

### DISCUSSION THREAD

**Discussion by Herbert-Sg** [breezy-weather/breezy-weather#1247] [Q&A] [answered]
I have signed up for the weather warning notification. But I can't tell from the warning which location it applies to. How can I find out?

**Comment by papjul** [comment, marked as answer]
Notifications are only sent for the first location.
You can know that by going to the main page of the settings. Subtitle of Notifications card states that it applies to the first location.
It's also written [in the documentation](https://github.com/breezy-weather/breezy-weather/blob/main/docs/UPDATES.md).

Also, if I remember correctly, the name of the location should be written somewhere in the notification, but it's possible it's not visible in older Android versions.

**Reply by Herbert-Sg** [reply]
Thank you for the information and sorry that I overlocked it.
